src/core/public/application/application_service.tsx

```diff
@@ -218,7 +224,7 @@ export class ApplicationService {
 
     const navigateToApp: InternalApplicationStart['navigateToApp'] = async (
       appId,
-      { path, state, replace = false }: NavigateToAppOptions = {}
+      { path, state, replace = false, openInNewTab }: NavigateToAppOptions = {}
```

Comment:
NIT: `openInNewTab = false`

Reply:
Updated in [`c1d3f3f` (#96723)](https://github.com/elastic/kibana/pull/96723/commits/c1d3f3f0cf8e8258c02a393daa71c74ca161fc46)